docs/reference/changelog.md

```diff
@@ -6,6 +6,11 @@
 
 * [#217](https://github.com/open-contracting/infrastructure/issues/217) - remove repeated 'OCDS:' in mapping documentation
 
+### Codelists
```

Comment:
Please update the changelog to fit with #276 (there'll be a merge conflict once either is merged)

Reply:
I'll update the changelog once #276 is merged.